### ISSUE TITLE
[Feature Request] uae/whdload new version

### ISSUE THREAD

**Issue by elessar38** [libretro/libretro-uae#569]
Can you please support uae 4.10 and whdload 18.8?

**Comment by sonninnos** [comment]
Nothing prevents you from updating WHDLoad under `saves/WHDLoad`, but I won't do that yet since it breaks some games.

Newer UAE version will come soonish once it is certain that it also won't break more things that it will fix.

**Comment by elessar38** [comment]
Rather, there is no perfect version of WHD or uae. Probably each will break one game while fixing another. It would be great to see the current versions of both uae and whd on Retroarch.
If I understood correctly, I can update the whdload myself in Retroarch. Unfortunately, I have no idea how to do it, even though I have a partition under standalone winuae 4.10 with the latest whdload. There are many files distributed in different folders. Is there any way you could upload these files to whdload update in retroarch for people who would like to have the latest version?

**Comment by sonninnos** [comment]
No. I will update it only after there is a new version that serves a net benefit, as in not break every single game that saves floppy disk sized save data.

Simply replace the `WHDLoad` executable in `saves/WHDLoad/C/` with the new one. It will stay as long as the directory is intact. When the whole `WHDLoad` directory is deleted it will update to the one baked in the core at the next launch.

**Comment by elessar38** [comment]
I checked whdload 18.8 and actually managed to catch the first error quickly. When selecting a character, there is an error in saving files in Ambermoon. I must add that the problem only occurs in retroarch, I have not encountered such problems when using os3.9 with whdload 18.8 and winuae 4.10.

**Comment by sonninnos** [comment]
Probably because this core sets `NoWriteCache` so that writes are done immediately and not at WHLoad quit, which most likely never happens when user quits RA.

**Comment by sonninnos** [comment]
Closing because the user is an entitled cunt. UAE updates when it updates, and WHDLoad updates when it no longer breaks the games in question.

**Comment by elessar38** [comment]
I sent a report that you are constantly insulting me. I wish you good luck.

**Comment by sonninnos** [comment]
FAFO.
